scripts/entrypoint.sh

```diff
@@ -85,5 +86,5 @@ else
     git add "${SC_SCORES_JSON}"
     git commit -m "Automated deployment to update ${SC_TARGET} static files $(date '+%Y-%m-%d')"
     git push origin "${UPDATE_BRANCH}"
-    /bin/bash -e /pull_request.sh
+    /pull_request.sh
```

Comment:
Just curious, why not leave /bin/bash to explicitly show the shell being used?

Reply:
From the commit message:

> These scripts already declare their interpreter in the shebang;
> specifying a shell or options is generally discouraged because it pokes
> into the implementation details.

<https://github.com/vsoch/cred-action/pull/4/commits/87bbc83e6ae58185111392998a539e02660ffdb2>

To elaborate slightly: it would be valid to refactor the script in
question to use `/bin/sh` instead of `/bin/bash`, or vice versa, or to
use `csh` or something else, or to not expect to be run with `set -e` in
effect and therefore use other patterns around exit statuses. All of
these would break when explicitly running with `/bin/bash -e`.